sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/commands/AbstractDetectorCommandImpl.java

```diff
@@ -94,6 +97,26 @@ public final Integer call() throws Exception {
     return handle(project, functions);
   }
 
+  private static boolean isVersionUnsupported(String salesforceApiVersion) {
+    boolean versionIsUnsupported = true;
+    return parseMajor(salesforceApiVersion)
+        .map(
+            targetVersion ->
+                parseMajor(DEFAULT_SALESFORCE_API_VERSION)
+                    .map(minimumVersion -> targetVersion < minimumVersion)
+                    .orElse(versionIsUnsupported))
+        .orElse(versionIsUnsupported);
```

Comment:
These two confused me at first. I think that's because the use of a local, mutable, variable signalled to me that the value can change. I think using `true` there instead isn't that bad and makes it clear that the default is constant. For types with more values than `boolean`, I would argue adding a proper constant is the way to go, but adding a constant for a `boolean` seems excessive to me.